### PR TITLE
Enterprise Blockchain Technologies - A University Course on Hyperledger Technologies

### PR DESCRIPTION
**Pull request by RafaelAPB**
Build a university course on Hyperledger technologies, with emphasis on Hyperledger Fabric.

### CODE REVIEW

**Comment by RafaelAPB** [comment]
Part of the 2020 Hyperledger Internship: https://wiki.hyperledger.org/display/INTERN/Build+a+university+course+on+Hyperledger+Fabric+using+Hyperledger+Umbra

**Comment by trbs** [comment]
Think you forgot to commit or push some things...

**Comment by trbs** [comment]
@dhuseby could you please confirm sponsorship ?

**Comment by dhuseby** [comment]
@trbs I confirm sponsorship. I am @RafaelAPB 's mentor for this project.

**Comment by dhuseby** [comment]
The course outline already exists in Gdocs and there are four of us collaborating on it already with more on the way. This repo is going to be the best place for us to collaborate. It will use another HL Lab--Hyperledger Umbra--as the simulation tool used for running the demonstrations and lab assignments.

**Comment by RafaelAPB** [comment]
@trbs can be, although other  Hyperledger technologies are tackled as well.

**Comment by trbs** [comment]
> @trbs could be an option, but we are tackling other Hyperledger technologies as well. If we only include Fabric on the title, I'm afraid it could be misleading.

I am sorry this is not the impression I get from reading the project description. 

There is nothing wrong with focusing on HL Fabric and I am happy to put my approval in but I do think it should either by `university-course-fabric` or the scope of the lab should be expanded.

**Comment by RafaelAPB** [comment]
@trbs ok - I've updated the project title.

File: labs/university-course.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Lab Name
-Enterprise Blockchain Technologies - A University Course on Hyperledger Technologies
+Enterprise Blockchain Technologies - A University Course on Hyperledger Fabric
```
Comment (trbs) [inline labs/university-course.md:2]: The lab name in the proposal should reflect the markdown file and repository name. (This is the common standard across other submissions as well.)

As per https://wiki.hyperledger.org/display/labs; the proposal name should be appropriate (e.g., not too generic, confusing, or conflicting with other labs or projects, etc)

I would still suggest naming the labs: `University Course Fabric` (file: labs/university-course-fabric.md)

If the group name of the coursework is `Enterprise Blockchain Technologies` I could imaging putting that in the short description.

Comment (RafaelAPB) [reply labs/university-course.md]: That makes sense, thanks for your input! I've updated the proposal.

**Comment by RafaelAPB** [comment]
@vipinsun as far as I know, there is no repo regarding this course.

**Comment by trbs** [comment]
@RafaelAPB looks like your good to go 👍 
Thanks for your patience and enjoy the labs 😃

**Comment by vipinsun** [comment]
I guess that there is no repo, but just gdocs. I merged the PR.

**Pull request merged**